lib/utils.js

```diff
@@ -55,34 +55,68 @@ exports.splitPop = (input, splitter) => {
 };
 
 /**
- * Gets plugins from node_modules and other directories
+ * Get plugin directories from global config
  *
- * @param {object} globalConfig - Configuration object
+ * @param {object} globalConfig - global configuration file contains plugin directories
  *
- * @returns {object} - all input plugins with path
+ * @returns {array} array of directory paths
  */
-exports.getPlugins = (globalConfig) => {
-  const existing = {};
+exports.getPluginsDirs = (globalConfig) => {
   const root = exec('npm root').toString().trim();
-  let dir = _.get(globalConfig, 'content.plugins.directory', []);
+  const core = _.get(globalConfig, 'content.plugins.core.directory', []);
```

Comment:
* `content.plugins.directories.core` and `content.plugins.directories.local` (something like that). Unless we have other sets of configuration for `core` and `local` they should be under a shared `directory` object.
* What is `core`? What is `local`? Content types should really only care about additional find paths for plugins above the normal `npm` path.